fenix/app/src/main/java/org/mozilla/fenix/compose/button/Button.kt

```diff
@@ -87,26 +94,33 @@ private fun Button(
  *
  * @param text The button text to be displayed.
  * @param modifier [Modifier] to be applied to the layout.
+ * @param enabled Controls the enabled state of the button.
+ * When false, this button will not be clickable
  * @param textColor [Color] to apply to the button text.
  * @param backgroundColor The background [Color] of the button.
  * @param icon Optional [Painter] used to display an [Icon] before the button text.
+ * @param iconModifier [Modifier] to be applied to the icon.
  * @param onClick Invoked when the user clicks on the button.
  */
 @Composable
 fun PrimaryButton(
     text: String,
     modifier: Modifier = Modifier.fillMaxWidth(),
+    enabled: Boolean = true,
     textColor: Color = FirefoxTheme.colors.textActionPrimary,
     backgroundColor: Color = FirefoxTheme.colors.actionPrimary,
     icon: Painter? = null,
+    iconModifier: Modifier = Modifier,
```

Comment:
I am also expecting the `iconModifier` param for the other buttons.

Reply:
Done